drift/lib/src/runtime/manager/ordering.dart

```diff
@@ -7,19 +7,16 @@ class ColumnOrderings<T extends Object> {
   /// It's used to expose ordering functions for a column
   ///
   /// {@macro manager_internal_use_only}
-  ColumnOrderings(this.column, {this.joinBuilders});
+  ColumnOrderings(this.column);
```

Comment:
Just to be sure I understood correctly - we can remove `joinBuilders` here because they're tracked in the top-level `ProcessedTableManager` instance now, right?

Reply:
Sorta, it's tracked by the `Composer` instead.

We used to pass the joins through the callback: 
```dart
ColumnOrderings order(f) {
   final ordering = f.id.asc();
   print(ordering.joinBuilders) // JoinBuilders are in this `ordering`
   return ordering;
}
```

Now we store it in `f`
```
ColumnOrderings order(f) {
   final ordering = f.id.asc();
   print(f.joinBuilders); // JoinBuilders are in this `f` Composer
   return ordering;
}
```
So Internally the building looks like:
```
// Create a brand new empty composer
final composer = createNewComposer();
print(composer.joinBuilders) // [] - It's empty

// Build the ordering, this will also add some joins the the composer
final ordering = order(composer);
print(composer.joinBuilders) // [JoinBuilder(...)] - Now it's full
```